internal/pkg/template/templates/cicd/buildspec.yml

```diff
@@ -2,9 +2,6 @@
 version: 0.2
 phases:
   install:
-    runtime-versions:
```

Comment:
if we do not specify any run-time versions, does code build choose default runtimes for the image?

Reply:
Yeah I think so